gulpfile.js

```diff
@@ -275,6 +296,9 @@ gulp.task('debug', ['dist', 'clean-debug'], function (done) {
                 process.exit(1);
             });
         }
+        var exec = require('child_process').exec;
+        console.log('Starting debug app...');
+        exec(getRunDebugAppCommand());
```

Comment:
I would probably store the command name in a variable, and then log something like `'Starting debug app (' + debugApp + ')...'`.